client/rest-high-level/src/main/java/org/elasticsearch/client/ml/dataframe/OutlierDetection.java

```diff
@@ -58,23 +60,27 @@ public static Builder builder() {
             }
             throw new IllegalArgumentException("Unsupported token [" + p.currentToken() + "]");
         }, METHOD, ObjectParser.ValueType.STRING);
+        PARSER.declareDouble(Builder::setMinScoreToWriteFeatureInfluence, MINIMUM_SCORE_TO_WRITE_FEATURE_INFLUENCE);
     }
 
     private final Integer nNeighbors;
     private final Method method;
+    private final Double minScoreToWriteFeatureInfluence;
 
     /**
      * Constructs the outlier detection configuration
      * @param nNeighbors The number of neighbors. Leave unspecified for dynamic detection.
      * @param method The method. Leave unspecified for a dynamic mixture of methods.
+     * @param minScoreToWriteFeatureInfluence The min outlier score required to calculate feature influence. Defaults to 0.1.
      */
-    private OutlierDetection(@Nullable Integer nNeighbors, @Nullable Method method) {
+    private OutlierDetection(@Nullable Integer nNeighbors, @Nullable Method method, @Nullable Double minScoreToWriteFeatureInfluence) {
         if (nNeighbors != null && nNeighbors <= 0) {
```

Comment:
I think we should either:
 a) add validation for the new field to be in [0, 1] interval
or
 b) remove validation for "nNeighbors".
in order to be consistent (either validate the user-provided input or not)

Reply:
Good point! I am in favour of keeping the validation logic in one place rather than duplicating it across. I'll remove the validation of `nNeighbors` from the client.